tools/feature_matrix.py

```diff
@@ -62,25 +63,44 @@ class Feature(IntEnum):
     'edge': 79,
     'firefox': 65,
     'safari': 120100,
-    # 'node': 120000
+    'node': 120000,
```

Comment:
This is a breaking change as the min node version disallows this, but all the others allow it I think.

Reply:
GLOBALTHIS / SUPPORTS_GLOBALTHIS is used only for AUDIOWORKLETS.  This means that AUDIOWORKLETS users are now more likely to see the message:

```
Must target recent enough browser versions that will support globalThis in order to target Wasm Audio Worklets
````

I rephrased to the message to be very clear what to the solution is:

```
AUDIO_WORKLET requires globalThis support, but this is disabled due to the current runtime configuration (most likely you need to build without `node` in ENVIRONMENT, or bump MIN_NODE_VERSION to >= 150000)
```